.github/workflows/release.yml

```diff
@@ -66,4 +66,25 @@ jobs:
           author: asyncapi-bot <info@asyncapi.io>
           title: 'chore(release): ${{ steps.extractver.outputs.version }}'
           body: 'Version bump in package.json and package-lock.json for release [${{ steps.extractver.outputs.version }}](https://github.com/${{github.repository}}/releases/tag/v${{ steps.extractver.outputs.version }})'
-          branch: version-bump/${{ steps.extractver.outputs.version }}
\ No newline at end of file
+          branch: version-bump/${{ steps.extractver.outputs.version }}
+  
+  dispatch:
+    needs: release
+    strategy:
+      matrix:
+        repo: ['asyncapi/generator']
```

Comment:
I'm 🤔 if this is the right way to do it, like maintain this matrix on a repo level instead of in central location. 
advantage of central location would be that this could be a separate workflow that reacts on `release` event, would be easy to maintan and update in all the other repositories 🤔

Reply:
Is this possible? Because honestly, I have no idea how to do it.

Reply:
we just need to decide where to store it, no GitHub Actions native functionality
we could have it as a yaml file in `.github` repository for example.

it is still not the best anyway, still pretty manual though

Reply:
So you mean, every repo will publish an event to every other repo —even if they don't have any kind of dependency between them— and then, on the other side, we only listen for events coming from the dependencies?